airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -349,17 +289,22 @@ def check_connection(self, logger, config) -> Tuple[bool, any]:
             return False, "Failed to authenticate with the provided credentials"
 
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
-        auth = self.initialize_authenticator(config)
-        api_endpoint = self.get_api_endpoint(config)
+        auth = initialize_authenticator(config)
+        api_endpoint = get_api_endpoint(config)
+        start_date = get_start_date(config)
 
         initialization_params = {
             "authenticator": auth,
-            "url_base": api_endpoint,
+            "url_base": api_endpoint
+        }
+        initialization_params_with_start_date = {
+            **initialization_params,
+            "start_date": start_date
         }
 
         return [
             Groups(**initialization_params),
-            Logs(**initialization_params),
+            Logs(**initialization_params_with_start_date),
```

Comment:
start_date should be passed to every incremental stream

Reply:
Add start date to all streams, because some full refresh streams use incremental streams to build path.